llvm/test/CodeGen/AMDGPU/strict_fpext.ll

```diff
@@ -1,22 +1,47 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 2
 ; FIXME: Missing operand promote for f16
```

Comment:
Still need the FIXME?

Reply:
Should have been removed in the previous commit to add the promotion handling